### PR TITLE
DO NOT MERGE : Implements SAIC-180 : evacuation script

### PR DESCRIPTION
**Pull request by oleksii-shyman**
- script reads series of actions from db and performs
    novaclient.servers.live_migrate call for each action

### CODE REVIEW

**Comment by Wasapl** [comment]
Build finished. Test PASSed.

**Comment by Wasapl** [comment]
Build finished. Test PASSed.

File: cloudferrylib/os/compute/nova_compute.py [hunk 1]
```diff
@@ -654,3 +654,15 @@ def get_free_disk(self):
         return (hypervisor_statistics.local_gb *
                 self.config.compute.disk_allocation_ratio -
                 hypervisor_statistics.local_gb_used)
+
+    def live_migrate_vm(self, vm_id, hostname):
+        LOG.debug("migrating {vm} to {host}".format(
+            vm=vm_id,
+            host=hostname
+        ))
+        self.nova_client.servers.live_migrate(
```
Comment (roman-verchikov) [inline cloudferrylib/os/compute/nova_compute.py]: Looks like live migration fails if VM has volume attached, so we should consider detaching volume prior to migration and attaching it back once it's migrated.

**Comment by roman-verchikov** [comment]
This was superseded with https://github.com/MirantisWorkloadMobility/CloudFerry/pull/208

**Pull request closed without merging**